src/components/SectionTitle/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './sectionTitle.css';
+
+const SectionTitle = ({ text }) => (
+  <div className="divider">
+    <div className="hr" />
```

Comment:
Whenever I find myself using classes that are named closely to native html elements it's usually a code smell that I could instead use the native element instead. Do you think we could use an `hr` here?